iocore/net/UnixNetVConnection.cc

```diff
@@ -1394,8 +1394,10 @@ UnixNetVConnection::migrateToCurrentThread(Continuation *cont, EThread *t)
   hold_con.move(this->con);
   SSLNetVConnection *sslvc = dynamic_cast<SSLNetVConnection *>(this);
 
-  SSL *save_ssl = (sslvc) ? sslvc->ssl : nullptr;
+  SSL *save_ssl                         = (sslvc) ? sslvc->ssl : nullptr;
+  const char *hold_ssl_client_cert_name = "";
   if (save_ssl) {
+    hold_ssl_client_cert_name = sslvc->options.ssl_client_cert_name;
```

Comment:
Why not entire `options`? Is this the only option that need to be preserved?

Reply:
Yes, this is the only value that is checked during the origin session match.  The sni name name is pulled from the SSL object directly.

Reply:
Hmm, are you suggesting I should add lines like this if I needed values in other fields? What would be a problem if we copied entire `options`?

I was surprised that this function copies only a few part of information in a NetVC. I'd be confused if I printed information in a NetVC for debugging after a migration.

Reply:
I guess just micro performance optimization.  Will update to copy all the options.